spec/models/book_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Book do
+  describe 'validations' do
+    it {should validate_presence_of :title}
+    it {should validate_presence_of :year}
+    it {should validate_presence_of :page_count}
+  end
+
+  describe 'relationships' do
+    it {should have_many :bookauthors}
+    it {should have_many :authors}
+    xit {should have_many :reviews}
```

Comment:
Is this test supposed to be skipped?

Reply:
I skipped it for now as we are not beginning work on review functionality.